source/utilities.d.ts

```diff
@@ -3,3 +3,7 @@ export type UpperCaseCharacters = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H'
 export type WordSeparators = '-' | '_' | ' ';
 
 export type StringDigit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9';
+
+export type StandartBuildInTypes =
+  | Function
+  | Date;
```

Comment:
Not sure we should include `Date` up front, lets get the PR merged without it at first, then it won't be able to cause any side-effects and we can get this feature out for you asap.

Reply:
> Not sure we should include Date up front

Why not? I cannot imagine anyone would want to camel-case the `Date` methods.

Reply:
This is similar to https://github.com/sindresorhus/type-fest/pull/208. Would be better to come up with a general solution than adding `Date`, `RegExp`, etc, explicitly.

Reply:
I guess `Date` is unique enough to include, but I wouldn't include `Function` in here I think, as `Date` is a built in `interface`, whereas `Function` isn't an interface?

We can't add eg. `Boolean` though I think, because of: https://github.com/sindresorhus/type-fest/issues/326#issuecomment-976913486